sass/themes/shop/2019/_typography.scss

```diff
@@ -0,0 +1,11 @@
+// Sailec regular font override with correct line height adjustments
+@font-face {font-family: 'Sailec-Regular';
+src: url('https://d1hcgerbie49io.cloudfront.net/webfonts/sailec-regular.eot');
+src: url('https://d1hcgerbie49io.cloudfront.net/webfonts/sailec-regular.eot?#iefix') format('embedded-opentype'),
+url('https://d1hcgerbie49io.cloudfront.net/webfonts/sailec-regular.woff2') format('woff2'),
+url('https://d1hcgerbie49io.cloudfront.net/webfonts/sailec-regular.woff') format('woff'),
+url('https://d1hcgerbie49io.cloudfront.net/webfonts/sailec-regular.ttf') format('truetype');
+font-display: fallback;}
+
+$font-stack: 'Sailec-Regular', sans-serif;
+
```

Comment:
I assume that the font used in the shop doesn't come from our css?

Reply:
Should've tied up some of these files; it looks like this file was never actually being imported in the original Shop theme either (all typography comes from base then SR18, and now CR19), so will remove